helm/secrets-provider/ci/test-values-template.yaml

```diff
@@ -0,0 +1,37 @@
+# Values for secrets-provider Chart. All missing values need to be supplied via environment variable.
```

Comment:
Why do you need 3 files for the values?

Reply:
To account for different scenarios 1. where I want to check defaults 2. override defaults 3. override image defaults

I wanted to avoid using `EXPORT var1=var` to prep in each test

Reply:
are these values form custom-values.yaml ?